src/main/java/org/spongepowered/api/network/NetworkPlayer.java

```diff
@@ -0,0 +1,43 @@
+package org.spongepowered.api.network;
+
+import org.spongepowered.api.entity.living.player.gamemode.GameMode;
+import org.spongepowered.api.entity.living.player.tab.TabListEntry;
+import org.spongepowered.api.profile.GameProfile;
+import org.spongepowered.api.text.Text;
+
+import java.util.Optional;
+
+/**
+ * @see TabListEntry
+ */
+public interface NetworkPlayer {
```

Comment:
Maybe NetworkPlayerEntry? NetworkPlayer itself may imply something different to others.